internal/core/api.rs

```diff
@@ -121,6 +121,39 @@ impl Default for CloseRequestResponse {
     }
 }
 
+pub use crate::input::PointerEventButton;
+
+/// A mouse or touch event
+#[derive(Debug, Clone, Copy, PartialEq)]
+#[allow(missing_docs)]
+pub enum PointerEvent {
+    /// The mouse or finger was pressed
+    Pressed { position: euclid::Point2D<f32, LogicalPx>, button: PointerEventButton },
+    /// The mouse or finger was released
+    Released { position: euclid::Point2D<f32, LogicalPx>, button: PointerEventButton },
+    /// The position of the pointer has changed
+    Moved { position: euclid::Point2D<f32, LogicalPx> },
+    /// Wheel was operated.
+    /// `pos` is the position of the mouse when the event happens.
+    /// `delta` is the amount of pixel to scroll.
+    Wheel { position: euclid::Point2D<f32, LogicalPx>, delta: euclid::Vector2D<f32, LogicalPx> },
```

Comment:
This is not criticism to your patch, just general observation: It's a little weird how we have verbose for the above variants, but we (and everyone else) is using a noun then. It could also be "WheelRotated", because that is what it us.